app/assets/javascripts/error.js

```diff
@@ -0,0 +1,22 @@
+pages.error.glassMovement = function(){
+  if ($('body').width() > 880) {
+    $('.error-img').parallax({
+      yparallax: '25px',
+      xparallax: '25px',
+      xorigin: 0,
+      yorigin: 0.9
+    });
+  }
+  $(window).on('resize', function(){
+    if ($('body').width() > 880) {
+      $('.error-img').parallax({
+        yparallax: '25px',
+        xparallax: '25px',
+        xorigin: 0,
+        yorigin: 0.9 });
+    } else {
+      $('.error-img').parallax({ yparallax: false, xparallax: false, });
+      $('.error-img').css({ 'left': '50%', 'marginLeft': '-125px' })
+    }
+  });
+};
```

Comment:
Amm not sure, but I think you could refactor and reuse the code for the parallax ... I mean, the if is the same, the target is the same, the parallax parameters are the same ... there should be a way to refactor....